### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by AndreaCrotti** [weavejester/clojure-toolbox.com#297]
I just noticed a broken link, and since the whole list of projects is in single yaml file, I thought we could simply scan then automatically.

Could be done in any language really, but I just wrote a few lines of Python:

```
import yaml, requests

for k, v in d.items():
    resp = requests.get(v['url'])
    if resp.status_code != 200:
        print("{} not found".format(k))

speclj not found
lein_diagnostics not found
sayid not found
```

Could it be automated I think as part of the build process?

**Comment by weavejester** [comment]
Having a script to check dead links is a good idea. Automating it is a touch harder, because you don't want to delete a project if its website is only temporarily down. I think I'd want to make actually removing dead links a manual process.

**Comment by AndreaCrotti** [comment]
Yes well I think the script should not remove things automatically, simply run periodically to check for broken links and send a notification for example.
Not sure how to do that as part of this project though, it's a bit tricky.
